runner/build.rs

```diff
@@ -0,0 +1,37 @@
+//
+// Copyright 2020 The Project Oak Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+use std::env::consts;
+
+fn main() {
+    let dsrt;
+    let debrt;
```

Comment:
What do these names mean? Could you add a comment / reference or rename them to something more explicit?

edit: I realised only now that they are the initals of the respective env vars, that was quite obscure and not very idiomatic in Rust :P Could you give them appropriate names, even if they are long? Also please explain in a comment why this approach is necessary.

Reply:
*facepalm* once again, guilty of no comments because I moved on to testing and forgot to note to make sure they were explained...